rfcs/050-concepts-api/README.md

```diff
@@ -0,0 +1,200 @@
+
+*   We're not trying to replicate Wikidata -- the APIs shouldn't return large amounts of information we aren't going to use on wellcomecollection.org.
+
+*   Any new APIs should be similar to the existing works and images APIs.
+    We'll use the same conventions for filters, aggregations, including optional fields, and so on.
+
+
+
+## Endpoint #1: retrieve a single concept
+
+This is the single concept endpoint, which is used to render a page for an individual subject or person:
+
+```
+GET /concepts/azxzhnuh
+{
+  "id": "azxzhnuh",
+  "identifiers": [
+    {
+      "identifierType": "lc-names",
+      "value": "n12345678",
+      "type": "Identifier"
+    },
+    ...
+  ],
+
+  "label": "Florence Nightingale",
+  "alternativeLabels": [
+    "The Lady with the Lamp",
+  ]
+
+  "type": "Person|Subject|Organisation|Place"
+
+  // Everything above here is stuff I'm pretty sure we'll need;
+  // everything below it is more nebulous and more likely to change
+  // in the final API.
+
+  "description": "Florence Nightingale was an English social reformer, statistician and the founder of modern nursing. Nightingale came to prominence while serving as a manager and trainer of nurses during the Crimean War, in which she organised care for wounded soldiers at Constantinople.",
+
+  // not locations
+  "urls": [
+    "https://en.wikipedia.org/wiki/Florence_Nightingale"
+  ],
+
+  // cf productionEvent?
+  "dates": [ { date, meaning } ]
+  "places": [ { place, meaning } ]
+  "birthDate/place"
+  "deathDate/place"
+
+  "thumbnail": { … },
+
+  "connectedConcepts": [
+    {
+      "id": "asoiham1",
+      "label": "Crimea",
+      "type": "Place"
+    }
+  ]
+}
+```
+
+Notes:
+
+-   The `id` and `identifiers` fields are the same as on existing works/images.
+
+-   The `label` and `alternativeLabels` fields are named analogously to `title` and `alternativeTitles` on works.
+
+-   The `type` will be an enumeration that includes Person, Organisation, Meeting, Subject.
+    For the initial implementation we'll likely have a few subtypes of Person but not of Subject.
+
+    The alternative is to have `type: Concept` and do subtypes through something similar to workType/format, but I'm not sure that adds anything.
+
+    This is a value users would be able to filter on (if we provide an endpoint for listing/searching concepts).
+
+    Although conceptually we know that Person/Organisation/Meeting are subtypes of Agent, the API doesn't.
+    For example, if you wanted to present a list of agent-like things, you'd use a filter `type=Person,Organisation,Meeting` rather than `type=Agent`.
+
+-   The `description` should provide enough context to help users distinguish subjects/people with the same label but different meaning.
+    This will likely be drawn from LCSH or Wikidata initially, but at some point we might write our own content here.
+
+    Q: Is `description` the best name for this field, or should it be called something different?
```

Comment:
I think it's fine, it matches Wikidata and we don't intend to write lots of long descriptions. That would be the only potential sticking point I see though, what would we call that if we did so?